frontend/e-buddy/src/components/chatbot/ChatEmbedGenerator.tsx

```
@@ -0,0 +1,84 @@
+              Copy and paste this code into your website where you want the chat widget to appear.
+            </p>
+            <textarea
+              readOnly
+              className="w-full h-40 p-2 border border-gray-300 rounded mb-4 font-mono text-sm"
+              value={embedCode}
+            />
+            <div className="flex justify-end space-x-2">
+              <button
+                onClick={copyToClipboard}
+                className="px-4 py-2 bg-indigo-500 hover:bg-indigo-600 text-white rounded"
+              >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)